app/jobs/email_checkin_reminder_job.rb

```diff
@@ -0,0 +1,30 @@
+class EmailCheckinReminderJob < ReservationJob
+  include EmailJobHelper
+  queue_as :default
```

Comment:
So are you going to move it into the parent job class and the others and remove it from the child classes?